apps/qubit/modules/actor/templates/_advancedSearch.php

```diff
@@ -26,7 +26,7 @@
                 <option value="not"<?php echo $item['operator'] == 'not' ? ' selected="selected"' : '' ?>><?php echo __('not') ?></option>
               </select>
 
-              <input class="query" type="text" placeholder="<?php echo __('Search') ?>" name="sq<?php echo $key ?>" value="<?php echo $item['query'] ?>"/>
+              <input class="query" type="text" placeholder="<?php echo __('Search') ?>" name="sq<?php echo $key ?>" value="<?php echo esc_entities($item['query']) ?>"/>
```

Comment:
Will calling `esc_entities()` explicitly here cause ampersands to be double-escaped (e.g. "&amp;amp;") when global escaping is enabled?

Reply:
Yeah, it looks like it ...

http://sandbox.onlinephpfunctions.com/code/4400d1eb438ac3b99720829979e958d2b38fcd3c